Flepper.QueryBuilder/FlepperQueryBuilder.cs

```diff
@@ -122,138 +122,16 @@ public static ISelectCommand SelectWithDistinct(params SqlColumn[] columns)
-        /// Create QueryBuilderParamenter Long
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueLong(long? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter ULong
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueULong(ulong? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter Float
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueFloat(float? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter Double
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueDouble(double? value = null)
-            => NullValue(value);
-
-
-        /// <summary>
-        /// Create QueryBuilderParamenter Char
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueChar(char? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter Bool
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueBool(bool? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter String
+        /// Create QueryBuilderParamenter
         /// </summary>
         /// <param name="value"></param>
         /// <returns></returns>
         public static IQueryBuilderParameter NullValueString(string value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter Decimal
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueDecimal(decimal? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter DateTime
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueDateTime(DateTime? value = null)
-            => NullValue(value);
-
-        /// <summary>
-        /// Create QueryBuilderParamenter TimeSpan
-        /// </summary>
-        /// <param name="value"></param>
-        /// <returns></returns>
-        public static IQueryBuilderParameter NullValueTimeSpan(TimeSpan? value = null)
-            => NullValue(value);
+            => new QueryBuilderParameter<char>(value);       
```

Comment:
char? for string?

Reply:
You requested a constraint for "Struct" and the solution was to pass a char. Do you have any solutions?

Reply:
Other solution:

https://github.com/Flepper/flepper/pull/90/commits/d8dbfb77d49040907615030b8b45c0a8c62cbfc9